### ISSUE TITLE
bug secondario travelbook, packhorse e flag fast travel forbidden

### ISSUE THREAD

**Issue by GiovGiorgio** [uomars/UOMars#835]
Salve, ecco un altro bug(?):
se nel packhorse ho item flaggati, starto il travelbook a piedi (quindi non mi appare il gump degli oggetti) e durante il conto alla rovescia salgo sulla cavalcatura, il contenuto flaggato del packhorse scompare completamente. è effettivamente un problema secondario, ma un giocatore disattento potrebbe rosikarci molto

**Comment by morpheus8888** [comment]
credo di non aver capito, cioè tu quando starti il travel con il packhorse, anche se non lo cavalchi non ti esce il gump? ti dovrebbe uscire comunque

**Comment by GiovGiorgio** [comment]
no, non esce il gump. e gli oggetti, anziché venire droppati a terra, si deletano
